packages/neutron-sdk/src/lib.rs

```diff
@@ -10,10 +10,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub use types::ProtobufAny;
-pub mod msg;
-pub mod query;
-pub mod types;
+pub mod bindings;
+mod errors;
+pub mod interchain_queries;
+pub mod interchain_txs;
+pub mod sudo;
+
+pub use errors::error::{NeutronError, NeutronResult};
```

Comment:
why not simply pub mod errors?

Reply:
just to make an exported API a little bit cleaner and structured?

Reply:
but the only pub things in the errors are NeutronError and NeutronResult. or maybe pub use errors::error (if it is a valid syntax)

Reply:
Because I want imports look like `use neutron_sdk::NeutronResult;`, not like `use neutron_sdk::errors:error:NeutronResult`. Them same with your comment above about `register_queries`

Reply:
okay then, thanks!

Reply:
> or maybe pub use errors::error (if it is a valid syntax)

then maybe `pub use errors::error::*`, but I don't like it